app/liquid_tags/user_subscription_tag.rb

```diff
@@ -1,9 +1,242 @@
 class UserSubscriptionTag < LiquidTagBase
+  PARTIAL = "liquids/user_subscription".freeze
+
+  SCRIPT = <<~JAVASCRIPT.freeze
+    function isUserSignedIn() {
+      return document.head.querySelector('meta[name="user-signed-in"][content="true"]') !== null;
+    }
+
+    // Hiding/showing elements
+    // ***************************************
+    function clearSubscriptionArea() {
+      document.getElementById('subscription-signed-in').style.display = 'none';
```

Comment:
When I start working on this, I'll refactor this so that instead of manipulating styles in this Javascript, we add CSS classes that control its styling.

Reply:
Love that! 💯